README.md

```diff
@@ -61,10 +61,10 @@ struct CreatePostRequest: NetworkRequest {
 }
 ```
 
-#### Option 2 - Using the `AnyNetworkRequest<T>` struct
+#### Option 2 - Using the `AnyRequest<T>` struct
 
 ```swift
-let createPostRequest = AnyNetworkRequest<Post>(method: .post,
+let createPostRequest = AnyRequest<Post>(method: .post,
                                                 url: URL(string: "http://jsonplaceholder.typicode.com/posts")!,
```

Comment:
For readability, let's get these parameters lined up.

Reply:
Fixed